### ISSUE TITLE
"Comments on Webviews and Web Content" draft section

### ISSUE THREAD

**Issue by jha11y** [w3c/matf#296]
Here is my proposed draft for this new section. 

It is a little bit lengthy, but the gist is:

Evaluate web content in mobile apps using WCAG 2.2 not this Group Note and the app creators are responsible for the accessibility of any web content because its inclusion is by choice. 

I used [Comments on Closed Functionality from WCAG2ICT](https://www.w3.org/TR/wcag2ict-22/#comments-on-closed-functionality as a starting point (mostly the first paragraph.)

I will read the comment and try and update the draft in this issue description accordingly so as not to make this a super long thread,

Begin Draft
-----------------
### h2. Comments on Webviews and Web Content ###

This Group Note assumes the presence of a “user agent” such as a browser application or browser-like application as a means to access and present web content in any mobile app using one or more webviews. Many of the success criteria in WCAG 2.2 assume web content will be accessed by these "user agents" which are installed on devices where assistive technologies can be connected to it or installed on it. This web content presented via any webview could be part of a mobile web app (an app that contains a single webview, which presents all of its content), a hybrid app (an application that contains both native mobile components and at least one webview) or a component in a mobile application that contains one or more webviews. The assistive technologies then present the web content from more or more webviews to people with disabilities in an accessible form.

Because WCAG 2.2 already provides normative and non-normative recommendations for making web content accessible, that standard and not this Group Note should be used for the evaluation of the accessibility of web content provided in any type of mobile app, inclusive. This is of particular importance because any web content presented via a webview is HTML-based, it cannot be directly affected by and is not a part of the native code of a mobile app. Any accessibility improvements made to web content presented through a webview must be made in the underlying HTML code. Due to the two points above, this Group Note will focus on recommendations for making the native mobile content accessible. No new recommendations for web content will be provided in this Group Note.

Native mobile applications do not require web content to function. The inclusion of web content is a choice, therefore, the creators (developers, designers, testers, managers, leadership, etc.) of a mobile app are fully responsible for ensuring the accessibility of all included web content.

### h3. NOTE ###

Examples of a browse application or a browser-like application include but are not limited to:
- Chrome app
- Safari app
- Android System Webview
- Webkit rendering engine
